docs/root/version_history/current.rst

```diff
@@ -79,6 +79,7 @@ New Features
 * listener: added ability to change an existing listener's address.
 * local_rate_limit_filter: added suppoort for locally rate limiting http requests on a per connection basis. This can be enabled by setting the :ref:`local_rate_limit_per_downstream_connection <envoy_v3_api_field_extensions.filters.http.local_ratelimit.v3.LocalRateLimit.local_rate_limit_per_downstream_connection>` field to true.
 * metric service: added support for sending metric tags as labels. This can be enabled by setting the :ref:`emit_tags_as_labels <envoy_v3_api_field_config.metrics.v3.MetricsServiceConfig.emit_tags_as_labels>` field to true.
+* stats: added native :ref:`Graphite-formatted tag <v1.18.0:envoy_v3_api_msg_config.metrics.v3.GraphiteStatsdSink>` support.
```

Comment:
hi @vaccarium - this wont work - its linking to the published docs - which dont include this - it should work with no prefix - ie linking to the current docs build - if its not it is more likely that the link itself is wrong

Reply:
..altho it looks correct to me

Reply:
it seems like the previous failure was due to `statssink` rather than `statsdsink`

` `Graphite-formatted tag <envoy_v3_api_msg_config.metrics.v3.GraphiteStatsdSink>` ` should work

Reply:
Thanks, I've removed the version prefix